flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/utils/OperationConverterUtils.java

```diff
@@ -144,6 +147,150 @@ public static Operation convertChangeColumn(
         // TODO: handle watermark and constraints
     }
 
+    public static Operation convertRenameColumn(
+            ObjectIdentifier tableIdentifier,
+            String originColumnName,
+            String newColumnName,
+            CatalogTable catalogTable,
+            ResolvedSchema originResolveSchema) {
+        Schema originSchema = catalogTable.getUnresolvedSchema();
+        List<String> tableColumns =
+                originSchema.getColumns().stream()
+                        .map(Schema.UnresolvedColumn::getName)
+                        .collect(Collectors.toList());
+        // validate old column is exists or new column is duplicated or old column is
```

Comment:
Nit:
```suggestion
        // validate old column is exists or new column isn't duplicated or old column is
```
And the following comments with `validate` should also change in same way.